canonical-websites/www.ubuntu.com
# 1587 support pricing

## Done

Updated the /support/plans-and-pricing page to VBT

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)


## Issue / Card

Fixes #1587 

## [Demo](http://www.ubuntu.com-1587-support-pricing.demo.haus/support/plans-and-pricing)
